esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -39,7 +39,7 @@ The |RTW| performs the following steps:
   :Executes:
      The |ESMValTool| command line script
   :Details:
-     Runs for every metric defined in the workflow
+     Runs for every group of recipes defined in the workflow
```

Comment:
When I run it, I get a process_heatwaves_coldwaves and a process_ensclus which to me means that process runs for every recipe rather than once per group.

Reply:
Thanks @mo-tgeddes - both comments should be addressed by 5923eea2810c010319d46aa06a198bdea1f82b95